foundation/application_test.go

```diff
@@ -282,6 +282,15 @@ func (s *ApplicationTestSuite) TestMakeLang() {
 }
 
 func (s *ApplicationTestSuite) TestMakeLog() {
+	mockConfig := &mocksconfig.Config{}
```

Comment:
```suggestion
	mockConfig := mocksconfig.NewConfig()
```

Use the New* method, we don't need to call `AssertExpectations` again.